includes/class-ssp-frontend.php

```diff
@@ -362,34 +362,38 @@ public function episode_meta_details ( $episode_id = 0, $context = 'content' ) {
 		$meta = apply_filters( 'ssp_episode_meta_details', $meta, $episode_id, $context );
 
 		$meta_display = '';
+		$podcast_display = '';
+		$subscribe_display = '';
+
 		$meta_sep = apply_filters( 'ssp_episode_meta_separator', ' | ' );
 		foreach ( $meta as $key => $data ) {
 
 			if( ! $data ) {
 				continue;
 			}
 
-			if( $meta_display ) {
-				$meta_display .= $meta_sep;
+			if( $podcast_display ) {
+				$podcast_display .= $meta_sep;
 			}
-
+			
 			switch( $key ) {
 
 				case 'link':
-					$meta_display .= '<a href="' . esc_url( $data ) . '" title="' . get_the_title() . ' " class="podcast-meta-download" download>' . __( 'Download file' , 'seriously-simple-podcasting' ) . '</a>';
+					$podcast_display .= '<a href="' . esc_url( $data ) . '" title="' . get_the_title() . ' " class="podcast-meta-download" download>' . __( 'Download file' , 'seriously-simple-podcasting' ) . '</a>';
 				break;
 
 				case 'new_window':
 					$play_link = add_query_arg( 'ref', 'new_window', $link );
-					$meta_display .= '<a href="' . esc_url( $play_link ) . '" target="_blank" title="' . get_the_title() . ' " class="podcast-meta-new-window">' . __( 'Play in new window' , 'seriously-simple-podcasting' ) . '</a>';
+					$podcast_display .= '<a href="' . esc_url( $play_link ) . '" target="_blank" title="' . get_the_title() . ' " class="podcast-meta-new-window">' . __( 'Play in new window' , 'seriously-simple-podcasting' ) . '</a>';
 				break;
 
 				case 'duration':
```

Comment:
I'm not sure I understand the reason this has been left in?

Reply:
Human/committer error here, doesn't show up in my local copy for some reason. Next commit should fix it.

Reply:
As discussed, please add the @todo comment, thanks